utils/pdf_to_text.sh

```diff
@@ -0,0 +1,3 @@
+java -jar $PDFBOX_HOME ExtractText $1 "$1.tmp"
```

Comment:
The arguments to this script should be documented, and it should have the ability to process an entire directory...

Reply:
Oof completely didn't mean to push the jar, nice catch. I'll squash it when we merge so that it's not accessible.

Reply:
It's pretty simple with just a for loop, I mostly just thought it's too much work for something that can be done externally easy.